### PR TITLE
Add deleteGameStreetAction

### PR DESCRIPTION
**Pull request by ryotogashi**

### CODE REVIEW

**Comment by ryotogashi** [comment]
It is going to be ready after #198 is merged.

**Comment by ryotogashi** [comment]
@axross Can you check in your spare time?

**Comment by axross** [comment]
@ryotogashi I have an improvement plan for test code. Could you let me know time we can do a pair-programming?

File: hooks/usePlayerActions/deleteGameStreetActionAt.test.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
+import { GameStreetActionType } from "../../models/GameSituation";
+import deleteGameStreetActionAt from "./deleteGameStreetActionAt";
+
+describe("deleteGameStreetActionAt( actions: GameStreetAction[], index: number ): GameStreetAction[]", () => {
+  it("throws error when index is equal to length of gameStreetActions", () => {
```
Comment (axross) [inline hooks/usePlayerActions/deleteGameStreetActionAt.test.ts:5]:
```suggestion
  it("throws an error when the index is equal to the length of gameStreetActions", () => {
```

**Pull request merged**